src/components/TouchableWithoutFocus.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {TouchableOpacity} from 'react-native';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** Element that should be clickable  */
+    children: PropTypes.oneOfType([
+        PropTypes.node,
+        PropTypes.func,
+    ]).isRequired,
+
+    /** Callback for onPress event */
+    onPress: PropTypes.func.isRequired,
+
+    /** Styles that should be passed to touchable container */
+    styles: PropTypes.arrayOf(PropTypes.object),
+};
+
+const defaultProps = {
+    styles: [],
+};
+
+/**
+ * This component prevents the tapped element from capturing focus
+ */
+class TouchableWithoutFocus extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onPress = this.onPress.bind(this);
+    }
+
+    onPress() {
+        if (!this.props.onPress) {
+            return;
+        }
+        this.touchableRef.blur();
+        this.props.onPress();
+    }
+
+    render() {
+        return (
+            <TouchableOpacity onPress={this.onPress} ref={e => this.touchableRef = e} style={this.props.styles}>
```

Comment:
```suggestion
            <TouchableOpacity onPress={this.onPress} ref={el => this.touchableRef = el} style={this.props.styles}>
```
This is the typical convention that we use for ref callbacks. `e` is usually used for an `event` argument

Reply:
Fixed